### PR TITLE
Treasury payment/fines fix (hopefully)

### PR DESCRIPTION
**Pull request by BadAtThisGame302**
## About The Pull Request

I always spawn in the woods for some reason and I can't test it because even round start it spawns my Steward in the woods and the travel point is a dead end.... SO THIS HOPEFULLY WORKS!!!

## Why It's Good For The Game

Hopefully lets people see if they've been fined/paid so they stop fucking ASKING FUCK YOU WHYMAN

## Pre-Merge Checklist

- [ ] You tested this on a local server.
- [x] This code did not runtime during testing.
- [x] You documented all of your changes.

### CODE REVIEW

File: code/controllers/subsystem/rogue/treasury.dm [hunk 1]
```diff
@@ -147,24 +147,28 @@ SUBSYSTEM_DEF(treasury)
...
+/datum/controller/subsystem/treasury/proc/give_money_account(amt, target, source)
 	if(!amt)
 		return
-	if(!name)
+	if(!target)
 		return
+	var/target_name = target
+	if(istype(target,/mob/living/carbon/human))
+		var/mob/living/carbon/human/H = target
+		target_name = H.real_name
```
Comment (ThePainkiller) [inline code/controllers/subsystem/rogue/treasury.dm:158]: Cheeky code (I hate BYOND) but this might work

**Comment by NPC1314** [comment]
Lets give it a try then

**Review by Optimism333 — COMMENTED** [review]
Any feedback on whether this has worked or not?

**Comment by NPC1314** [comment]
Caused the bug where you cant tell who made a deposit or something like that, not fixed AFAIK

**Review by NPC1314 — APPROVED** [review]

**Pull request merged**